packages/blade/src/components/Box/BaseBox/BaseBox.native.tsx

```diff
@@ -1,7 +1,24 @@
+import { View } from 'react-native';
 import styled from 'styled-components/native';
+import { getBaseBoxStyles } from './getBaseBoxStyles';
 import type { BaseBoxProps } from './types';
-import getBaseBoxStyles from './getBaseBoxStyles';
 
-const BaseBox = styled.View<BaseBoxProps>(getBaseBoxStyles);
+/**
+ * Some prop go to React Native DOM and fail with type errors.
```

Comment:
we can. Although we are sending all props to dom right now. we do have issue to discuss that #714 so it will get fixed when we are resolving that issue